### ISSUE TITLE
Better handle accessibility support across browsers

### ISSUE THREAD

**Issue by bacloud14** [bacloud14/-WeatherVenue-Issues#2]

**Comment by lghiggino** [comment]
Hello. Which benchmarks are being used to evaluate accessibility support? Google lighthouse? Wave? Googles accessibility tools? I'd like to solve this issue

**Comment by bacloud14** [comment]
Hello @lghiggino 

Thank you for showing interest. Actually, the name "accessibility" as I used, can be very misleading. 
As intended at first, it is not really to offer the accessibility through "keyboard" as it seemed hard to implement (?)

The way I went through this, is to natively implement a simple command system through `SpeechSynthesisUtterance` and `SpeechRecognition`. It happens these APIs are not very supported and can change also unpredictably, also there is some polemic on user data and how it is used (because recognition offered by Chrome is online and not offline).
I am thinking of [this offline solution](https://syl22-00.github.io/pocketsphinx.js/live-demo.html#grammar-definitions) and I am willing to work on it.

To sum up, I am very open to make the website also open for accessibility through keyboard. You are very welcome to take a look on code.

**Comment by bacloud14** [comment]
you can see the same issue here https://github.com/bacloud14/WeatherVenue/issues/24 when you join the repo.

**Comment by bacloud14** [comment]
@lghiggino is this an issue you would like to work on ?
I sent you an invite to https://github.com/bacloud14/WeatherVenue/

**Comment by lghiggino** [comment]
Unfortunatelly I feel this implementation is, at the moment, above of what I could deliver. This issue could be better solved by someone else.

**Comment by bacloud14** [comment]
By the way, I meant I'm also considering implementing accessibility the traditional way. Like commanding webpages through keyboard.
Anyway, by all means, you are welcome for contributing.
